### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Port to 2.0
~~Added conditional check for Polymer.Base._warn existence (to be implemented https://github.com/Polymer/polymer/issues/3926)~~

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,17 +19,35 @@
   },
   "main": "paper-toast.html",
   "dependencies": {
-    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.0",
-    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#^1.0.9",
-    "iron-fit-behavior": "PolymerElements/iron-fit-behavior#^1.1.0",
-    "polymer": "Polymer/polymer#^1.5.0"
+    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#2.0-preview",
+    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#2.0-preview",
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
```
Comment (notwaldorf) [inline bower.json]: need to add `iron-fit-behavior` here too

Comment (valdrinkoshi) [inline bower.json]: `iron-overlay-behavior 1.x` depends on `iron-fit-behavior ^1.0.0`, while `paper-toast` requires `iron-fit-behavior 1.1.x` or superior ([this PR](https://github.com/PolymerElements/paper-toast/pull/74) introduced the dependency on this specific version).
In 2.x, `iron-overlay-behavior` depends on `iron-fit-behavior 2.x` so `paper-toast` doesn't need to specify that anymore.

Comment (notwaldorf) [inline bower.json]: Not sure I understand. This PR has the same code running for both Polymer 1.0 and Polymer 2.0. There's no code changes here so either neither version uses any direct code from `iron-fit-behaviour`, and this dependency isn't need, or both are, and then it's needed in both

Comment (valdrinkoshi) [inline bower.json]: paper-toast requires iron-fit-behavior 1.1.0 or superior.
iron-overlay-behavior 1.x does not enforce this (it has dependency on iron-fit-behavior 1.0.0), so the 1.x variant of paper-toast needs to specify that dependency.
Iron-overlay-behavior 2.x already satisfies paper-toast dependency on iron-fit-behavior.
If it helps it to be clearer, I can add iron-fit-behavior dependency also in the main variant

Comment (notwaldorf) [inline bower.json]: yeah, maybe? i think it doesn't hurt, and keeping it symmetric makes it...clearer nothing different is going on.

Comment (valdrinkoshi) [inline bower.json]: Done 👌

**Pull request merged**